### PR TITLE
PBM-742 Oplog replay

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/index.rst
new file:   source/oplog-replay.rst

### CODE REVIEW

**Review by defbin — APPROVED** [review]

File: source/oplog-replay.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.. _oplog-replay:
```
Comment (spron-in) [inline source/oplog-replay.rst:1]: Should we mention here somewhere that for now we still require a full snapshot?

Comment (defbin) [reply source/oplog-replay.rst:1]: oplog-replay itself does not require backup. only oplog. backup could be removed but the command still will work

Comment (defbin) [reply source/oplog-replay.rst:1]: but to start saving oplog, backup is needed as for now

**Review by defbin — APPROVED** [review]

File: source/oplog-replay.rst (hunk 1 above)
Comment (nastena1606) [reply source/oplog-replay.rst:1]: Added

File: source/oplog-replay.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+*******************
+|PITR| oplog replay
+*******************
+
+Starting with version 1.7.0, you can replay the :term:`oplog <Oplog>` for a specific period on top of any backup: logical, physical, storage level snapshot (like EBS-snapshot). In this way you can manually restore sharded clusters and non sharded replica sets to a specific point in time from a backup made by any tool and not only by |PBM|.
+
+.. warning::
+
+   Use the oplog replay functionality with caution, only when you are sure about the starting time to replay oplog from. However, be aware that data consistency is not guaranteed when replaying oplog on top of storage level snapshots and backups made not with |PBM|.
```
Comment (dAdAbird) [inline source/oplog-replay.rst:11]: In the second sentence, I'd rather state that in general PBM won't guarantee data consistency while using oplog reply despite the restore method. Though there are more chances to screw up with non-PBM backup, using it with PBM backup still leaves some room for bummer.

Comment (nastena1606) [reply source/oplog-replay.rst]: Amended

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**